pkg/api/core.go

```diff
@@ -24,26 +24,20 @@ import (
 	"net/http"
 	"strings"
 
-	"bytes"
-	"time"
-
 	"github.com/gorilla/mux"
-	"github.com/prometheus/common/expfmt"
 	"github.com/sapcc/maia/pkg/keystone"
 	"github.com/sapcc/maia/pkg/storage"
 
-	dto "github.com/prometheus/client_model/go"
-	"io"
+	"fmt"
+	"github.com/databus23/goslo.policy"
+	"github.com/sapcc/maia/pkg/util"
+	"github.com/spf13/viper"
+	"io/ioutil"
 )
 
-// RFC822 timestamp format
-const RFC822 = "Mon, 2 Jan 2006 15:04:05 GMT"
-
-var prometheusCoreHeaders = make(map[string]string)
-
 //VersionData is used by version advertisement handlers.
 type VersionData struct {
-	Status string            `json:"status"`
+	Status string            `json:"Status"`
```

Comment:
btw. in Grafana we used user@domain, not user@project. The project scope was set by switching the project in the UI